com.unity.shadergraph/Editor/Data/Nodes/UV/ParallaxOcclusionMappingNode.cs

```diff
@@ -22,8 +17,6 @@ public ParallaxOcclusionMappingNode()
             UpdateNodeAfterDeserialization();
         }
 
-        public override string documentationURL => Documentation.GetPageLink("SGNode-Parallax-Occlusion-Mapping");
```

Comment:
Shouldn't the documentation link be updated?

Reply:
As I understand this is only needed if node is outside shadergraph package

Reply:
correct, the root for nodes is set in the base class AMN, so it should be correct here